tuftool/README.md

```diff
@@ -109,6 +109,14 @@ tuftool update \
    --timestamp-version 2 \
    --outdir "${WRK}/tuf-repo" \
    --metadata-url file:///$WRK/tuf-repo/metadata
+
+# finally, downlaod tuf repo
```

Comment:
Per discussion elsewhere, this example could be given in another section. Or, if not, the comment should at least explain that the download command is usually used to download a remote repo with an HTTP/S URl.